### PR TITLE
Feature/figure improvements

### PR DESCRIPTION
**Pull request by quinnwai**
* Improves figure code
* Make changes missed by previous PR (see [02cd40d](https://github.com/ohsu-comp-bio/vrs-python-testing/pull/67/commits/02cd40d6aefaa4c8961813ada23983baa93f7039))
* Updates usage in the readme

There is an associated [figure notebook](https://app.terra.bio/#workspaces/terra-test-bwalsh/vrs-anvil/analysis/launch/1000g-figures.ipynb) on Terra that creates figures for all chrs and mirrors the 1000g-sample-stats.py file in this PR that does the same for only chr1, would be great if you could check if that runs too @bwalsh 

Addressing #38, #40, #60, #69

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,18 +57,19 @@ In addition, this project facilitates the retrieval of evidence associated with
...
 
 ### Usage
+To start, set up a manifest file ([example here](tests/fixtures/manifest.yaml)) in your working directory. We like to do with `tmp/` folder in the repo's root directory.
 
 **CLI**
-
 ```bash
 source venv/bin/activate
-vrs_anvil input.vcf
+vrs_anvil annotate
```
Comment (bwalsh) [inline README.md]: compare w/ the quickly drafted changes I made here.
https://github.com/ohsu-comp-bio/vrs-python-testing/pull/63/commits/2782e6766769d84cae4c967007538dbb0bcdf3fb

File: vrs_anvil/__init__.py [hunk 2]
```diff
@@ -356,19 +358,24 @@ def check_paths(self) -> "Manifest":
         return self
 
 
-def query_metakb(id, log=False):
+def query_metakb(vrs_id, log=False):
```
Comment (bwalsh) [inline vrs_anvil/__init__.py:307]: Maybe this belongs in the script?   Let's discuss.

File: scripts/1000g-sample-stats.py [hunk 3]
```diff
@@ -12,7 +12,6 @@
 """
```
Comment (bwalsh) [inline scripts/1000g-sample-stats.py:17]: At a high level, this looks good.   However, I get confused when I compare it with `1000g-figures.ipynb`

**Review by bwalsh — COMMENTED** [review]
Looks good.  A few issues to discuss. See below.

**Comment by bwalsh** [comment]
image

File: README.md (hunk 1 above)
Comment (quinnwai) [reply README.md]: updated

File: vrs_anvil/__init__.py (hunk 2 above)
Comment (quinnwai) [reply vrs_anvil/__init__.py:307]: I think it could stay in since it's agnostic to analysis and data will be pulled in anyways. If we want to add CAF creation to this repo in the future  this will also be needed

File: scripts/1000g-sample-stats.py (hunk 3 above)
Comment (quinnwai) [reply scripts/1000g-sample-stats.py:17]: yeah there are some discrepancies between the two, I added some initial commenting code to `scripts/1000g-sample-stats.py` so someone could run it locally as a test example of how to get 1000g figures from metrics created for a single vcf (chr1). The `1000g-figures.ipynb` is primarily for people to replicate our entire chromosome run on Terra.

Comment (quinnwai) [reply scripts/1000g-sample-stats.py:17]: Lmk this might be a good thing to add to the docs

File: README.md [hunk 4]
```diff
@@ -75,19 +71,27 @@ vrs_anvil annotate
...
 
+**Processing VCF Files ([vrs-python](https://github.com/ga4gh/vrs-python))**
 
-```
+For Python usage, see [vrs_vcf_annotator.py](scripts/vrs_vcf_annotator.py).
 
-**Processing VCF Files**
-TODO - update this section with vrs-python instructions...
-Replace `input.vcf` with the name of your VCF file. The processed VCF file with GA4GH VRS identifiers will be generated as `output_processed.vcf` in the same directory.
+For CLI usage, use something like
```
Comment (bwalsh) [inline README.md]: tighten up language.
s/For CLI usage, use something like/For example:/

File: README.md [hunk 5]
```diff
@@ -75,19 +71,27 @@ vrs_anvil annotate
...
 vrs_anvil annotate --scatter
+
+# run the vrs_anvil command in parallel in the background
+nohup vrs_anvil annotate --scatter & # press enter to continue
+
 # get the status of the scatter processes
 vrs_anvil ps
+```
 
+**Processing VCF Files ([vrs-python](https://github.com/ga4gh/vrs-python))**
```
Comment (bwalsh) [inline README.md:82]: Add a sentence or two explaining what this command does.

File: README.md [hunk 6]
```diff
@@ -75,19 +71,27 @@ vrs_anvil annotate
...
+```bash
+python3 -m ga4gh.vrs.extras.vcf_annotation --vcf_in tests/fixtures/1kGP.chr1.1000.vcf --vcf_out annotated_output.vcf.gz --vrs_pickle_out allele_dicts.pkl --seqrepo_root_dir ~/seqrepo/latest
+```
 
-**Developing in Python**
-For an example notebook on usage, see [1000g_patient_na12878_evidence.ipynb](scripts/1000g_patient_na12878_evidence.ipynb). If working on Terra, see the vrs-anvil workspace.
+The above is an example using an example vcf. Replace the `--vcf_out` and `vrs_pickle_out` here with your desired output file path, where the output vcf can be BCF (`vcf.gz`) or VCF (`vcf`)
 
+**Terra**
+All CLI works in Terra CLI as well. For an example notebook, see `vrs-anvil-demo.ipynb` on the `vrs-anvil` workspace.
```
Comment (bwalsh) [inline README.md]: tighten up language.  repeating CLI in the sentence

File: vrs_anvil/__init__.py (hunk 2 above)
Comment (bwalsh) [reply vrs_anvil/__init__.py:307]: ok

File: vrs_anvil/cli.py [hunk 7]
```diff
@@ -151,41 +187,56 @@ def ps_cli(ctx):
...
-            for _ in scattered_processes['processes']:
-                log_file = 'NA'
-                metrics_file = 'NA'
+            for _ in scattered_processes["processes"]:
+                log_file = "NA"
+                metrics_file = "NA"
                 try:
                     log_file = sorted(state_dir.glob(f"vrs_anvil*{_['pid']}.log"))[-1]
-                    metrics_file = sorted(state_dir.glob(f"metrics_*{_['pid']}.yaml"))[-1]
+                    metrics_file = sorted(state_dir.glob(f"metrics_*{_['pid']}.yaml"))[
```
Comment (bwalsh) [inline vrs_anvil/cli.py:208]: this is a confusing re-formatting

File: vrs_anvil/cli.py
Comment (bwalsh) [inline vrs_anvil/cli.py:1]: There seems to be a lot of re-formatting in this file.  Is this something that vscode is doing automatically? 
If we want formatting this way, it would be useful to do it in a separate commit.  That way, the lines do not need to be "manually" read as part of the review

**Review by bwalsh — CHANGES_REQUESTED** [review]

File: vrs_anvil/cli.py
Comment (quinnwai) [reply vrs_anvil/cli.py:1]: Yes, I believe it was the function of black `vrs_anvil/cli.py`.  On splitting it out into separate commits, I agree, do you have a good way to do that? I think I had it combined cause the pre-commit would only allow me to commit a given set of changes if it's formatted properly.

File: README.md (hunk 5 above)
Comment (quinnwai) [reply README.md:82]: done

File: README.md (hunk 4 above)
Comment (quinnwai) [reply README.md]: done

File: README.md (hunk 6 above)
Comment (quinnwai) [reply README.md]: done

**Comment by quinnwai** [comment]
Tested that:
- script runs in Terra
- [`1000g-figure.ipynb`](https://app.terra.bio/#workspaces/terra-test-bwalsh/vrs-anvil/analysis/launch/1000g-figures.ipynb?mode=edit) can create figures without having VCF or metrics files loaded in
- `1000g-figure.ipynb` can create figures with VCF or metrics files already loaded in

**Comment by bwalsh** [comment]
LGTM, please squash and merge

**Review by bwalsh — APPROVED** [review]
LGTM, please squash and merge
